### PR TITLE
Constants: Add Dymension to cosmwasm chains

### PR DESCRIPTION
**Pull request by barnjamin**
Adds Dymension chain to constants, requested by @nik-suri

### CODE REVIEW

File: sdk/js/src/cosmwasm/address.ts [hunk 1]
```diff
@@ -35,6 +36,7 @@ export const isNativeDenomNeutron = (denom: string) => denom === "untrn";
 export const isNativeDenomCelestia = (denom: string) => denom === "utia";
 export const isNativeDenomStargaze = (denom: string) => denom === "ustars";
 export const isNativeDenomSeda = (denom: string) => denom === "aseda";
+export const isNativeDenomDymension = (denom: string) => denom === "dym";
```
Comment (evan-gray) [inline sdk/js/src/cosmwasm/address.ts:39]: I think this is `udym`, according to the readme https://github.com/dymensionxyz/dymension?tab=readme-ov-file#initializing-dymd

**Review by evan-gray — CHANGES_REQUESTED** [review]

File: sdk/js/src/cosmwasm/address.ts (hunk 1 above)
Comment (barnjamin) [reply sdk/js/src/cosmwasm/address.ts:39]: good spot, poking around i found [here](https://github.com/dymensionxyz/dymension/blob/9fa05e7f1f0dfc9d6d83ecf3a309c24eda203bee/scripts/src/genesis_config_commands.sh#L82) is `adym` too, in code search on that repo `udym` is always in some `_test` file

Comment (nik-suri) [reply sdk/js/src/cosmwasm/address.ts:39]: Confirmed that this should be `adym` with the Dymension team.

Comment (barnjamin) [reply sdk/js/src/cosmwasm/address.ts]: updated this

**Review by evan-gray — APPROVED** [review]

**Comment by evan-gray** [comment]
> LGTM, thanks @barnjamin !
> 
> Can we also add this chain ID to the rust SDK constants in `chain.rs`?

maybe save this for a separate PR so as to bother those code owners with that change 🙂

**Review by SEJeff — APPROVED** [review]

**Pull request merged**